sdk/eventhub/event-processor-host/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "plugins": ["@azure/azure-sdk"],
-  "extends": ["../../.eslintrc.old.json", "plugin:@azure/azure-sdk/recommended"],
```

Comment:
Does every package that wants to use the plugin have to have its own .eslintrc.json file? I see that the storage packages dont in #11472

Reply:
Not necessarily. Some packages do have their own files to be more selective about which rules to turn on/off such as `core-paging`. I would like customer facing ones to eventually not have custom `.eslintrc.json`.